### PR TITLE
Add helper script to xip-patch

### PR DESCRIPTION
**Pull request by Demonsthere**
**Description**

Changes proposed in this pull request:

- Add helper script

**Related issue(s)**

Needed in #2563

### CODE REVIEW

File: components/xip-patch/certhelper.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    fi
+done
+
+if [ "${discoverUnsetVar}" = true ] ; then
+    exit 1
+fi
+
+if [[ -z "${DOMAIN}" ]]; then
+	echo "---> DOMAIN not SET. Creating..."
+	INGRESS_IP=$(getLoadBalancerIP "${LB_SERVICE_NAME}" "${LB_SERVICE_NS}")
```
Comment (Tomasz-Smelcerz-SAP) [inline components/xip-patch/certhelper.sh]: This can result in an error. Handle the error.

Comment (sjanota) [reply components/xip-patch/certhelper.sh]: Would `set -e` be enough?

Comment (Demonsthere) [reply components/xip-patch/certhelper.sh]: It is, look at line 1 ;)

**Pull request closed without merging**